### PR TITLE
Frontend/search

### PR DESCRIPTION
**Pull request by shiwangsingh84**

### CODE REVIEW

File: marktzeit/supermarkets/views.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+<<<<<<< HEAD
```
Comment (vil-s) [inline marktzeit/supermarkets/views.py]: looks like a merge error

File: config/settings/base.py [hunk 2]
```diff
@@ -40,8 +40,8 @@
 # DATABASES
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
-DATABASES = {"default": env.db("DATABASE_URL")}
-DATABASES["default"]["ATOMIC_REQUESTS"] = True
+DATABASES = {"default": "sqlite:///db.sqlite3"}
+# DATABASES["default"]["ATOMIC_REQUESTS"] = True
```
Comment (vil-s) [inline config/settings/base.py]: are these changes for local development?

File: config/settings/local.py [hunk 3]
```diff
@@ -43,11 +44,11 @@
...
-if env("USE_DOCKER") == "yes":
-    import socket
-
-    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
-    INTERNAL_IPS += [ip[:-1] + "1" for ip in ips]
+# if env("USE_DOCKER") == "yes":
+#     import socket
+#
+#     hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+#     INTERNAL_IPS += [ip[:-1] + "1" for ip in ips]
```
Comment (vil-s) [inline config/settings/local.py]: also for your local development?

File: config/settings/local.py [hunk 4]
```diff
@@ -56,3 +57,9 @@
 
 # Your stuff...
 # ------------------------------------------------------------------------------
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(APPS_DIR, 'db.sqlite3'),
+    }
+}
```
Comment (vil-s) [inline config/settings/local.py]: and here

**Review by vil-s — CHANGES_REQUESTED** [review]
Looks good, just some questions regarding the changes in the settings and a merge error

File: config/settings/local.py (hunk 4 above)
Comment (shiwangsingh84) [reply config/settings/local.py]: reverted

File: config/settings/local.py (hunk 3 above)
Comment (shiwangsingh84) [reply config/settings/local.py]: reverted

File: config/settings/base.py (hunk 2 above)
Comment (shiwangsingh84) [reply config/settings/base.py]: reverted

File: marktzeit/supermarkets/views.py (hunk 1 above)
Comment (shiwangsingh84) [reply marktzeit/supermarkets/views.py]: corrected

**Review by vil-s — APPROVED** [review]
LGTM

**Pull request merged**